### ISSUE TITLE
[0.0.118 > server]: No dimension set for key window

### ISSUE THREAD

**Issue by joncursi** [necolas/react-native-web#587]
After upgrading from 117 -> 118, my server is throwing:

```
Invariant Violation: No dimension set for key window
W20170801-10:58:22.222(-4)? (STDERR)     at invariant (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/fbjs/lib/invariant.js:44:15)
W20170801-10:58:22.222(-4)? (STDERR)     at Function.get (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/react-native-web/dist/apis/Dimensions/index.js:1:1414)
W20170801-10:58:22.223(-4)? (STDERR)     at Object.<anonymous> (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/redbird-core/lib/helpers/normalizeSize.js:13:44)
W20170801-10:58:22.223(-4)? (STDERR)     at Module._compile (module.js:569:30)
W20170801-10:58:22.224(-4)? (STDERR)     at Object.Module._extensions..js (module.js:580:10)
W20170801-10:58:22.225(-4)? (STDERR)     at Module.load (module.js:503:32)
W20170801-10:58:22.225(-4)? (STDERR)     at tryModuleLoad (module.js:466:12)
W20170801-10:58:22.225(-4)? (STDERR)     at Function.Module._load (module.js:458:3)
W20170801-10:58:22.226(-4)? (STDERR)     at Module.require (module.js:513:17)
W20170801-10:58:22.226(-4)? (STDERR)     at require (internal/module.js:11:18)
```

The code being executed that worked on 117 but is throwing on 118 is:

```
const {
  height,
  width,
} = Dimensions.get('window');
```

**Comment by necolas** [comment]
Thanks. Should be fixed by 8d80885f5d95b48f17b3e5d352de4fb42972fd01

**Comment by joncursi** [comment]
Indeed it is. Thank you!
